src/lightning_app/utilities/introspection.py

```diff
@@ -170,8 +170,6 @@ class LightningCallbackVisitor(LightningVisitor):
     methods: Set[str] = {
```

Comment:
There's one more occurrence for the app directory in https://github.com/Lightning-AI/lightning/blob/fbfcc3d871539340aa6d47f444c38726a7c536d8/src/lightning_app/cli/pl-app-template/core/callbacks.py#L251

@awaelchli Do you know what's the best way to change it?

Reply:
I knew this day would come :D 
This is not going to be an easy change to be included in this PR I think, as the purpose there is to collect the init args of trainer and the alternative would have to be more involved. So I need to follow up on this immediately. Creating an issue now.